documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
If the last encoding should be right in the end, do not merge it yet, some last corrections are almost ready